bitnami/cassandra/values.yaml

```diff
@@ -194,6 +194,9 @@ dbUser:
   forcePassword: false
   # password:
   # existingSecret:
+  #  name: mySecret
+  #  keyMapping:
+  #    cassandra-password: myKey
```

Comment:
I'm not sure we should add these sub-parameters like this. It can be confusing for people that already upgraded to this major, as it seems they are mandatory. A solution will be to add a comment explaining that it accepts both structures (only defining `existingSecret: secretname` and the new standard using common helpers). Please take a look how this is implemented at the bitnami/ghost chart's [values.yalm](https://github.com/bitnami/charts/blob/fafe2a30b1e46d1ed5f7b325e042d09d026ab801/bitnami/ghost/values.yaml#L99) and README.

You will also need to replicate any changes done in `values.yaml` on `values-production.yaml`

Reply:
You are right, it would only be in very specific cases where there could be confusion. Nevertheless, the backward compatibility comment takes care of it. Thanks!